packages/app/cypress/e2e/debug.cy.ts

```diff
@@ -58,9 +58,24 @@ describe('App - Debug Page', () => {
 
     cy.visitApp()
 
+    cy.intercept('mutation-SideBarNavigation_RecordEvent').as('recordEvent')
     cy.findByTestId('sidebar-link-debug-page').click()
     cy.findByTestId('debug-container').should('be.visible')
 
+    // TODO: comment back in once https://github.com/cypress-io/cypress/issues/26693 merges
```

Comment:
This entire test is really flaky locally, even before this addition. I believe this is improved significantly in #26693.